main.tex

```diff
@@ -0,0 +1,86 @@
+\documentclass{article}
+\usepackage[utf8]{inputenc}
+
+\title{LaTeX}
+\author{Kinga }
+\date{May 2019}
+\title{LaTeX Project}
+\author{Kinga Straszkiewicz}
+\usepackage[style=authoryear]{biblatex}
+\usepackage{ulem}
+\usepackage{tikz}
+\usepackage{tikz-qtree}
+\usepackage{tipa}
+\usepackage{phonrule}
+\addbibresource{my_biblio.bib}
+\date{April 2019}
+
+\begin{document}
+
+\maketitle\
+
+\section{Section1}
+\uwave {My interests:}
+\begin{enumerate}
+    \item the origin of language
+    \item phonology
+    \item phonetics
+    \item grammar
+    \item syntax
+    \item phonotactics
+    \item wordbuilding
+    \item programming
+    \item semantics
+\end{enumerate}
+\newline
+\section{Section2}
+\begin{IPA}
+\textipa{/"maI "neIm Iz "kIngA. "aI "aem @ "stju:d@nt @t @ jUnI"v3:s@ti @v "gdansk. "maI "Int@r@sts "Inklu:d lIn"gwIstIks, I"spES@li f@"n6l@dZi aend "graem@r, b@t "6lseU "6rIdZIn @v "laengwIdZ "aend "proUgraemIN./}
```

Comment:
1. Please eliminate periods, there is no punctuation in transcription!
2. a university? The university.
3. in'clude
4. linguistics - velar nasal
5. there is a symbol for ash in tipa
6. also - long vowel; the diphthong looks different too
7. language, and, programming - correct the ash; language has a velar nasal